docs/create_topology.md

```diff
@@ -78,28 +189,35 @@ Keysight controller yaml:
 kubectl apply -f ixiatg-operator.yaml
 ```
 
-> TIP: The IxiaTG controller does not have to be deployed manually, during
-> [cluster deployment](#deploy_a_cluster) the IxiaTG controller can be
-> automatically deployed if specified in the deployment yaml.
-
-### SR Linux
+### SR Linux Controller
 
 > IMPORTANT: Make sure to use the `kind-bridge.yaml` deployment config. This is
 > because the SR Linux controller + containers require the `bridge` cluster CNI
 > instead of the default `ptp` cluster CNI that `kind` uses.
 
-To manually apply the controller run the following command:
+To manually apply the controller, first clone the `srl-controller`
+[repo](setup.md#srl-controller) and then run the following command:
 
 ```bash
-kubectl apply -k https://github.com/srl-labs/srl-controller/config/default
+kubectl apply -k srl-controller/config/default
```

Comment:
Hi @alexmasi 
you don't need to clone the repo to install srl-controller.
Unless there was some hidden meaning you can say 

> To manually apply the controller run `kubectl apply -k https://github.com/srl-labs/srl-controller/config/default`

Reply:
Thanks, I accidentally changed this from our internal docs differing instructions. Reverted